src/app/Processor/ProcessorEdit/ProcessorEdit.css

```diff
@@ -32,6 +32,10 @@
   padding: var(--processor-edit--spacing);
 }
 
+.processor-edit__form__notice {
+  margin-top: var(--pf-global--spacer--3xl);
```

Comment:
why do you need this custom margin top?

Reply:
I added it mainly for actions configuration. When an action is selected, the list of fields for its parameters is displayed.

With this margin there's some space between the end of the form and the end of the page and it's easier to understand there are some fields to take care of.